src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -261,22 +261,28 @@ def compute_isi_violations(sorting_analyzer, isi_threshold_ms=1.5, min_isi_ms=0,
     Returns
     -------
     isi_violations_ratio : dict
-        The isi violation ratio described in [Hill]_.
+        The isi violation ratio.
     isi_violation_count : dict
         Number of violations.
 
     Notes
     -----
-    You can interpret an ISI violations ratio value of 0.5 as meaning that contaminating spikes are
-    occurring at roughly half the rate of "true" spikes for that unit.
-    In cases of highly contaminated units, the ISI violations ratio can sometimes be greater than 1.
+    The returned ISI violations ratio measures the approximate fraction of spikes in each
+    unit which are contaminted. This interpretation is good when the ratio is small, and
+    becomes worse as it grows. In cases of highly contaminated units, the ISI  violations
+    ratio can sometimes be greater than 1.
+
+    This method counts the number of spikes whose isi is violated. If there are three
+    spikes within `isi_threshold_ms`, the first and second are violated. Hence there are two
+    spikes which have been violated.  This is is contrast to `compute_refrac_period_violations`,
+    which counts the number of violations.
 
     References
     ----------
-    Based on metrics described in [Hill]_
+    Based on metrics originally implemented in Ultra Mega Sort [UMS]_.
 
-    Originally written in Matlab by Nick Steinmetz (https://github.com/cortex-lab/sortingQuality)
-    and converted to Python by Daniel Denman.
+    This implementation is based on one written in Matlab by Nick Steinmetz
```

Comment:
```suggestion
    This implementation is based on one of the original implementations written in Matlab by Nick Steinmetz
```

maybe something like this. Because if people follow this link the implementation has been changed so we probably want to make it clear that it was originally based on the matlab but has since diverged? What do you think?

Reply:
Yeah, sounds good. It's mainly there to give credit to Nick, and the new link to UMS is nearby.